client/ayon_houdini/plugins/publish/collect_usd_layers.py

```diff
@@ -38,12 +39,15 @@ def copy_instance_data(instance_src, instance_dest, attr):
     keys = attr.split(".")
     for i, key in enumerate(keys):
         if key not in src_data:
-            break
+            raise PublishError(
+                f"key '{key}' does not exist on the source instance."
+            )
```

Comment:
Why is this suddenly an error if previously this would `break`?

Reply:
Tbh, this was a wild guess based on the doc string. 
should I put the `break` back and update the doc string?

Reply:
I've reverted the changes in this function.

https://github.com/ynput/ayon-houdini/pull/190/commits/65249ccabf63925f7aa369ad001f550914ff2299